packages/flame/lib/src/text.dart

```diff
@@ -44,154 +94,149 @@ class TextConfig {
   /// The name you choose for the font family can be any name (it's not inside the TTF file and the filename doesn't need to match).
   final String fontFamily;
 
-  /// The [TextAlign] to be used when creating the [material.TextPainter].
-  ///
-  /// Beware: it's recommended to leave this with the default value of [TextAlign.left].
-  /// Use the anchor parameter to [render] to specify a proper relative position.
-  final TextAlign textAlign;
-
-  /// The direction to render this text (left to right or right to left).
-  ///
-  /// Normally, leave this as is for most languages.
-  /// For proper fonts of languages like Hebrew or Arabic, replace this with [TextDirection.rtl].
-  final TextDirection textDirection;
-
-  /// The height of line, as a multiple of font size.
-  final double? lineHeight;
-
-  final MemoryCache<String, material.TextPainter> _textPainterCache =
-      MemoryCache();
-
-  /// Creates a constant [TextConfig] with sensible defaults.
+  /// Creates a constant [TextPaint] with sensible defaults.
   ///
   /// Every parameter can be specified.
-  TextConfig({
-    this.fontSize = 24.0,
+  const TextPaintConfig({
     this.color = const Color(0xFF000000),
     this.fontFamily = 'Arial',
-    this.textAlign = TextAlign.left,
```

Comment:
Note to reviewers: You may notice that I removed the `textAlign` property from the configs.

I did this because I realized this property don't make much sense on a text renderer as it only has a position when rendering, there isn't any dimension, so there isn't anything to align really.

Also we already recommended to use the `Anchor` instead of the property, so I took a leap here and removed it entirely to avoid confusion.